group_clusters.go

```diff
@@ -153,10 +155,11 @@ func (s *GroupClustersService) AddCluster(pid interface{}, opt *AddGroupClusterO
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/group_clusters.html#edit-group-cluster
 type EditGroupClusterOptions struct {
-	Name               *string                             `url:"name,omitempty" json:"name,omitempty"`
-	Domain             *string                             `url:"domain,omitempty" json:"domain,omitempty"`
-	EnvironmentScope   *string                             `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
-	PlatformKubernetes *EditGroupPlatformKubernetesOptions `url:"platform_kubernetes_attributes,omitempty" json:"platform_kubernetes_attributes,omitempty"`
+	Name                *string                             `url:"name,omitempty" json:"name,omitempty"`
+	Domain              *string                             `url:"domain,omitempty" json:"domain,omitempty"`
+	EnvironmentScope    *string                             `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
+	PlatformKubernetes  *EditGroupPlatformKubernetesOptions `url:"platform_kubernetes_attributes,omitempty" json:"platform_kubernetes_attributes,omitempty"`
+	ManagementProjectID *string                             `url:"management_project_id,omitempty" json:"management_project_id,omitempty"`
```

Comment:
According to the docs, it's not a valid field for this API call.

Reply:
The docs are incomplete but the API call works.